### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Isaac-Leonard** [esp-rs/esp-generate#287]
Would it be possible to specify features with commandline flags to avoid the TUI menu for selecting them?

I am blind and the menu doesn't work too well with my screen reader making it difficult to select the features I want or to know what features I have selected or what is selected by default.
If there is already a way to do this, could it be put in the readme as I could not find a way to do so from a quick read over it.

**Comment by Isaac-Leonard** [comment]
Actually just checking the readme again it appears as if this already exists, though perhaps more documentation could be added for the specific flags to use for each feature.
Either way, I'll close this for now and leave it up as a tribute to reading the docs more closely before raising an issue.

**Comment by bugadani** [comment]
We are definitely open to any suggestions that could make your experience better!

**Comment by Isaac-Leonard** [comment]
Actually looking at the readme even the docs are fine I reccon, think I'm just having a slow day.

**Comment by Isaac-Leonard** [comment]
The one I use for the terminal is at https://github.com/tspivey/tdsr though I may not be using the most recent commit.
Also I can't seem to get -o or -o defmt to work with the CLI command as the readme and result of running esp-generate --help says I should be able to.

**Comment by bugadani** [comment]
Thank you. I've reopened this issue so that we have a reminder. At the very least we will need to look into the documented options. We need to check what we can do about dependencies between options that the TUI handles, but the CLI options may not.
